ballerina/build.gradle

```diff
@@ -85,7 +85,8 @@ task clean {
 }
 
 task build {
-    build.dependsOn ':qm-ballerina:qm:build'
+    build.dependsOn ':qm-ballerina:api_inspectionlot_srv:build'
+    build.dependsOn ':qm-ballerina:updateDocumentationFiles'
```

Comment:
https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.procurement/blob/main/ballerina/build.gradle#L88

Reply:
Updated here [Update docs.json,settings.gradle and build.gradle for PR build](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/11/commits/187ec88523c7b17afc72a2961fecad9d88601941)